config/config.toml.example

```diff
@@ -143,6 +143,9 @@ cross-join = true
 # Stats lease duration, which influences the time of analyze and stats load.
 stats-lease = "3s"
 
+# Whether run stats updates on this tidb-server. It will updates stats only when stats-lease is non-zero.
```

Comment:
```suggestion
# Whether run stats updates on this tidb-server. It will use the value of `stats-lease` to set its tickers' values, so it won't work then `stats-lease` is zero.
```